structure_tests/drivers/tar_driver.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 Google Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package drivers
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	pkgutil "github.com/GoogleCloudPlatform/container-diff/pkg/util"
+	"github.com/GoogleCloudPlatform/runtimes-common/structure_tests/types/unversioned"
+)
+
+type TarDriver struct {
+	Image pkgutil.Image
+}
+
+func NewTarDriver(imageName string) (Driver, error) {
+	// try the local daemon first. if no dice, try a remote registry
+	var prepper pkgutil.Prepper
+	prepper = pkgutil.DaemonPrepper{
+		Source: imageName,
+	}
+	var image pkgutil.Image
```

Comment:
nit: you can inline this with the if err line below

Reply:
Actually, I'm not sure the tar driver needs to support daemon images, if your image is in a daemon you can just run the tests there, right? The main use case for this will be images that are already in a tar, or in a registry and you want to run the tests without having a daemon.

Reply:
yeah good call, we only need to support remote images here. updated